ct-ng menuconfig fails, macos high sierra

I can't get **_ct-ng menuconfig_** to work on current master.
 commit fd9fe523b22cb6281f26081232a3f8f3aee7fda1 and newer gives:
`  CONF  menuconfig
config/configure.in:48:warning: ignoring unsupported character '@'
config/configure.in:49:warning: ignoring unsupported character '@'
config/configure.in:50: syntax error
config/configure.in:49: unknown option "KCONFIG_install_with_strip_program"
make: *** [menuconfig] Error 1`

previous commit 59b08f8d06700770cf1105269acb2ec2460e2a34 works fine

host: macos High Sierra 10.13.3
[config.log](https://github.com/crosstool-ng/crosstool-ng/files/1858069/config.log)

--- a/Makefile.in
+++ b/Makefile.in
@@ -44,7 +44,7 @@ __silent_rmdir = $(call __silent,RMDIR,$1)rm -rf $1
 # Remove any suffix rules
 .SUFFIXES:
 
-all: Makefile build
+all: configure Makefile build
 
 ###############################################################################
 # Configuration variables
@@ -102,6 +102,10 @@ PROG_NAME := $(shell echo 'ct-ng' |$(sed) -r -e '$(PROG_SED)' )
 # Check if Makefile is up to date:
 Makefile: Makefile.in
 	@echo "$< changed: you must re-run './configure'"
+	@false
+
+configure: configure.ac
+	@echo "$< changed: you must re-run './bootstrap'"
 	@false
 
 # If installing with DESTDIR, check it's an absolute path

